lib/travis/build/git/clone.rb

```diff
@@ -40,6 +40,7 @@ def checkout
           def clone_args
             args = "--depth=#{depth}"
             args << " --branch=#{branch}" unless data.ref
+	    args << " #{quiet_o}"
```

Comment:
why not remove the need for the `quiet_o` method and just add it similar to how the `--branch` is added?

also, looks like the indenting is a little off